crates/pathfinder/src/core.rs

```diff
@@ -143,14 +143,36 @@ pub struct GlobalRoot(pub StarkHash);
 #[derive(Copy, Clone, PartialEq, Deserialize, Serialize)]
 pub struct StarknetBlockHash(pub StarkHash);
 
+macro_rules! i64_masquerading_as_u64_newtype_to_from_sql {
```

Comment:
I named it this, because I realized that ToSql of u64 uses `i64::try_from` which means u64 can only be 0..=i64::MAX.

Reply:
Added.